x-pack/plugins/fleet/public/applications/integrations/sections/epm/constants.tsx

```diff
@@ -86,7 +86,7 @@ export const AssetTitleMap: Record<DisplayedAssetType, string> = {
   csp_rule_template: i18n.translate(
     'xpack.fleet.epm.assetTitles.cloudSecurityPostureRuleTemplate',
```

Comment:
It's really minor but we should probably also change the id to `benchmarkRuleAssetTitle`. if you decide to do that we'll probably also need to run `node scripts/i18n_check.js --fix` after that

Reply:
will take a look into it, thanks!

Reply:
@JordanSh I quickly checked and found that `cloudSecurityPostureRuleTemplate` is also key in `KibanaAssetType` and `KibanaSavedObjectType` enums. Renaming the i18n key I guess is independent of these, but we'll lose the connection between the assets themselves and their UX text representation.`cloudSecurityPostureRuleTemplate` still makes sense in my opinion as an asset name when not exposed to the end user. wdyt?

Reply:
yeah thats makes sense 👌